### PR TITLE
Updated the experience page with dark mode

### PR DESCRIPTION
**Pull request by Tejas1510**
I have updated the experience page by adding dark mode feature

### CODE REVIEW

File: awesome-portfolio-websites/index.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+<head>
+    <title>John Doe | Home</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="keywords"
+        content="free portfolio template,portfolio template, portfolio website template, open source, open sourcing, contributing to open source" />
+    <meta name="description"
+        content="This project serves free and beautiful portfolio template for everyone. Anyone can showcase their work including moocs, education, projects, experience,etc. to the world." />
+    <!-- Robot.txt -->
```
Comment (smaranjitghose) [inline awesome-portfolio-websites/index.html:12]: Why is this file changed?

File: awesome-portfolio-websites/projects.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+<head>
+  <title>John Doe | Projects</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="keywords"
+    content="free portfolio template,portfolio template, portfolio website template, open source, open sourcing, contributing to open source" />
+  <meta name="description"
+    content="This project serves free and beautiful portfolio template for everyone. Anyone can showcase their work including moocs, education, projects, experience,etc. to the world." />
+  <!-- Robot.txt -->
```
Comment (smaranjitghose) [inline awesome-portfolio-websites/projects.html:12]: Why is this file changed?

File: awesome-portfolio-websites/research.html [hunk 3]
```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (smaranjitghose) [inline awesome-portfolio-websites/research.html:3]: Why is this file changed?

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by Tejas1510** [comment]
Sir i haven't changed any of the above mentioned files
I just made some changes in experience.html,added a folder name dark-mode in assets folder which will be used for darkmode toggling and just pushed the entire repository back

**Comment by shreyakapoor08** [comment]
> Sir i haven't changed any of the above mentioned files
> I just made some changes in experience.html,added a folder name dark-mode in assets folder which will be used for darkmode toggling and just pushed the entire repository back

@Tejas1510 Please make sure you only push the code in which you have worked. No other files should be altered.
I suggest you to take pull from upstream first and also remove all the unnecessary changes.

**Comment by shreyakapoor08** [comment]
@Tejas1510 Also resolve the conflicts. Let me know if you need any help

**Comment by Tejas1510** [comment]
@shreyakapoor08  Mam can you please guide how can i delete all the files that i have pushed and push just the required ones

**Comment by Tejas1510** [comment]
It is showing a conflict in experience.html in which the content that i have added so how to resolve it

**Comment by shreyakapoor08** [comment]
Message me on slack please
